h2o-core/src/main/java/hex/tfidf/DocumentFrequency.java

```diff
@@ -0,0 +1,50 @@
+package hex.tfidf;
+
+import water.fvec.Frame;
+import water.rapids.ast.prims.mungers.AstGroup;
+
+/**
+ * Task class using map-reduce to compute document frequency values for words in given documents.
+ */
+public class DocumentFrequency {
+
+    /**
+     * Column names to be used for output frame.
+     */
+    private static final String[] OUTPUT_FRAME_COL_NAMES = new String[] { "Word", "DF" };
+
+    /**
+     * Computes document frequency values for given words in documents.
+     *
+     * @param wordFrame input frame of words for which document frequency
+     *                  values should be computed. For exact format
+     *                  see {@link TermFrequency}.
+     *
+     * @return frame containing document frequency values for given words.
+     */
+    public Frame compute(Frame wordFrame) {
```

Comment:
can be static

Reply:
Good point. I changed it for both `TermFrequency` and `DocumentFrequency` and in places where `compute` method is being used.